embedding-calculator/src/docs/find_faces_base64_post.yml

```diff
@@ -0,0 +1,91 @@
+tags:
+  - Core
+summary: 'Find faces in the given image in BASE64 format and return their bounding boxes.'
+description: 'Returns bounding boxes of detected faces on the image.'
+operationId: findFacesBase64Post
+consumes:
+  - multipart/form-data
+produces:
+  - application/json
+parameters:
+  - in: formData
+    name: file
+    type: file
+    required: 'true'
+    description: 'A picture with at least one face in BASE64 format.'
+  - in: formData
```

Comment:
I believe it should be `query`, not `formData`
And yes, it looks like some parameters in `find_faces_post.yml` also defined wrong. I would be glad if you fix there too

Reply:
Fixed